apps/src/lib/levelbuilder/code-docs-editor/ProgrammingExpressionsTable.jsx

```diff
@@ -0,0 +1,251 @@
+import React, {useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import * as Table from 'reactabular-table';
+import queryString from 'query-string';
+import Button from '@cdo/apps/templates/Button';
+import StylizedBaseDialog from '@cdo/apps/componentLibrary/StylizedBaseDialog';
+import PaginationWrapper from '@cdo/apps/templates/PaginationWrapper';
+import CloneProgrammingExpressionDialog from './CloneProgrammingExpressionDialog';
+
+const DEFAULT_VALUE = 'all';
+
+export default function ProgrammingExpressionsTable({
```

Comment:
I think it would be useful to have a header comment that gives a description of the component to help orient someone who might not be familiar with the code.